apps/meteor/app/authentication/server/startup/index.js

```diff
@@ -196,7 +196,7 @@ const onCreateUserAsync = async function (options, user = {}) {
 	if (!user.active) {
 		const destinations = [];
 		const usersInRole = await Roles.findUsersInRole('admin');
-		await usersInRole.toArray().forEach((adminUser) => {
+		(await usersInRole.toArray()).forEach((adminUser) => {
```

Comment:
this is actually what they tried to do =)
```suggestion
		await usersInRole.forEach((adminUser) => {
```